### DISCUSSION TITLE
how to create the dataset for training??

### DISCUSSION THREAD

**Discussion by sumith5** [mindee/doctr#1775] [Q&A]
Hi Guys,
Can you guide me for the custom dataset preparation like which annotation tool need to be used to get the required format for training the text detection model.
Thanks in advance for help

**Comment by felixdittrich92** [comment]
Hi @sumith5 :wave:,

Currently we don't have a custom labeling tool for docTR so you can use any available of your choice (for example: [LabelStudio](https://github.com/HumanSignal/label-studio) and afterwards convert the labels to the required format:

[docs](https://mindee.github.io/doctr/using_doctr/custom_models_training.html)

Reference datasets can be found [here](https://github.com/mindee/doctr/discussions/1654)

Additional a great blog post about fine tuning docTR models: [blog post](https://www.christianmoscardi.com/blog/2024/10/03/digitizing-us-statistical-abstracts.html)

Hope this helps

Best,
Felix :)

**Reply by sumith5** [reply]
Hello @felixdittrich92,

Thank you for your prompt response.

I am working on building a custom OCR model for a new dataset. The dataset is annotated in COCO format, with a single class labeled as "text." My objective is to perform OCR on these images.
can you guide me for the conversion of the annotations in the required format and also please feel free to share your opinion   
 
Please find below a sample image from my dataset for reference.
![sample](https://github.com/user-attachments/assets/394ee4cb-638e-4a56-82cd-07f8c7256a7f)
